ElasticBox/appliance-openstack.md

```diff
@@ -7,12 +7,30 @@
 }}}
 
 ### Installing the Virtual Appliance in OpenStack
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.